public/js/app/ladderWeek.js

```diff
@@ -0,0 +1,124 @@
+/*
+ https://api.faforever.com/leaderboards/ladder1v1?page[size]=50&page[number]=1
+ */
+ 
+const categories = JSON.parse(rankingCategories);
+
+var getPage = function(category, pageSize, id) {
+  $("#navigation-first-"+category).addClass("disabled");
+  $("#navigation-previous-"+category).addClass("disabled");
+  $("#navigation-next-"+category).addClass("disabled");
+  $("#navigation-last-"+category).addClass("disabled");
+  if (currentPage[category] < lastPage[category]){
+      $("#navigation-next-"+category).removeClass("disabled");
+      $("#navigation-last-"+category).removeClass("disabled");
+  }
+  if (currentPage[category] > 1){
+      $("#navigation-first-"+category).removeClass("disabled");
+      $("#navigation-previous-"+category).removeClass("disabled");
```

Comment:
extract method for

`$("#navigation-XXXXXXX-"+category).removeClass("disabled");`